File: fibo/c/fibo.c
#include <stdio.h>

int fibo(int n) {
    if (n == 0) {
        return 0;
    } else if (n == 1) {
        return 1;
    } else {
        return fibo(n - 2) + fibo(n - 1);
    }
}

int main() {
    printf("%d\n", fibo(40));
}

File: fibo/erl/fibo.erl
-module(fibo).
-export([start/0]).

fibo(0) -> 0;
fibo(1) -> 1;
fibo(N) -> fibo(N-1) + fibo(N-2).

start() ->
  io:format("~w\n", [fibo(40)]),
  halt().
